ros/processor/inventory_events_consumer.py

```diff
@@ -131,6 +131,7 @@ def process_system_details(self, msg):
                     "cloud_provider": host['system_profile']['cloud_provider'],
                     "stale_timestamp": host['stale_timestamp'],
                     "operating_system": host['system_profile']['operating_system'],
+                    "groups": host['groups']
```

Comment:
Do we always get the `groups`field under the `host` object? I would prefer to use `get()` method.
```suggestion
                    "groups": host.get('groups')
```

Reply:
@r14chandra we do always get the `groups` field under the host object but I still updated the code to use `get()` method as I feel that will be better. I will push the changes in the latest code.